common/flatpak-run.c

```diff
@@ -1834,7 +1840,10 @@ get_dconf_data (const char  *app_id,
   client = dconf_client_new ();
 
   g_debug ("Add defaults in dir %s", prefix);
-  add_dconf_dir_to_keyfile (defaults_data, client, prefix);
+  add_dconf_dir_to_keyfile (defaults_data, client, prefix, DCONF_READ_DEFAULT_VALUE);
```

Comment:
Why do we want/need to copy default values? Might as well just insert the minimum subset of user values that are changed?